.rubocop.yml

```diff
@@ -7,6 +7,7 @@ AllCops:
   DisplayCopNames: true
   Exclude:
     - 'db/**/*'
+    - 'lib/traject/**/*'
```

Comment:
Please remove this.  I don't want to skip rubocop for all things in lib/traject.